schema/aiconfig.schema.json

```diff
@@ -157,15 +157,32 @@
           "type": "object",
           "additionalProperties": {},
           "properties": {
-            "data": {
-              "description": "Input to the model. This can represent a single input, or multiple inputs.\nThe structure of the data object is up to the ModelParser. Attachments field should be leveraged for non-text inputs (e.g. image, audio)."
-            },
             "attachments": {
-              "description": "Used to include non-text inputs (e.g. image, audio) of specified MIME types in the prompt",
+              "description": "Attachments can be used to pass in non-text inputs (ex: image, audio)",
               "type": "array",
               "items": {
-                "$ref": "#/definitions/Attachment"
+                "type": "object",
+                "properties": {
+                  "data": {
+                    "description": "The data representing the attachment"
+                  },
+                  "mime_type": {
+                    "description": "The MIME type of the result. If not specified, the MIME type will be\nassumed to be text/plain",
+                    "type": "string"
+                  },
+                  "metadata": {
+                    "description": "Attachment metadata",
+                    "type": "object",
+                    "additionalProperties": {}
+                  }
+                },
+                "required": [
+                  "data"
+                ]
               }
+            },
+            "data": {
+              "description": "Input to the model. This can represent a single input, or multiple inputs.\nThe structure of the data object is up to the ModelParser. For example,\na multi-modal ModelParser can choose to key the data by MIME type."
```

Comment:
Why the change to the descriptions here? I think it's pretty confusing to reference mime-types here instead of having an opinionated stance on using attachments for different mime type data -- reading these descriptions I wouldn't know which to pick and it wouldn't seem to matter?

Reply:
I think we didn't update the schema using the script for a while. I've updated the description to remove references to MIME types